client/src/Movies/Movie.js

```diff
@@ -2,67 +2,73 @@ import React, { Component } from 'react';
 import axios from 'axios';
 
 export default class Movie extends Component {
-  constructor(props) {
-    super(props);
-    this.state = {
+	constructor(props) {
+		super(props);
+		this.state = {
       movie: null
+      
     };
-  }
+    console.log(this.state.movie)
+	}
 
-  componentDidMount() {
-    // change this line to grab the id passed on the URL
-    const id = 1;
-    this.fetchMovie(id);
+	componentDidMount() {
+		// change this line to grab the id passed on the URL
+		const id = this.props.match.params.id;
+		this.fetchMovie(id);
   }
+  
 
-  fetchMovie = id => {
-    axios
-      .get(`http://localhost:5000/api/movies/${id}`)
-      .then(response => {
-        this.setState(() => ({ movie: response.data }));
-      })
-      .catch(error => {
-        console.error(error);
-      });
-  };
-  // Uncomment this code when you're ready for the stretch problems
-  // componentWillReceiveProps(newProps){
-  //   if(this.props.match.params.id !== newProps.match.params.id){
-  //     this.fetchMovie(newProps.match.params.id);
-  //   }
-  // }
 
-  // saveMovie = () => {
-  //   const addToSavedList = this.props.addToSavedList;
-  //   addToSavedList(this.state.movie)
-  // }
+	fetchMovie = id => {
+		axios
+			.get(`http://localhost:5000/api/movies/${id}`)
+			.then(response => {
+				this.setState(() => ({ movie: response.data }));
+			})
+			.catch(error => {
+				console.error(error);
+			});
+	};
+	// Uncomment this code when you're ready for the stretch problems
+	componentWillReceiveProps(newProps) {
+		if (this.props.match.params.id !== newProps.match.params.id) {
+			this.fetchMovie(newProps.match.params.id);
+		}
+	}
 
-  render() {
-    if (!this.state.movie) {
-      return <div>Loading movie information...</div>;
-    }
+	saveMovie = event => {
```

Comment:
Because you're not actually doing anything with the `event` variable, in this case, you can remove it.